extensions/wikia/Email/Email.i18n.php

```diff
@@ -3,6 +3,7 @@
 
 $messages['en'] = array(
 	'emailext-desc' => 'An extension to handle delivering email',
+	'emailext-anonymous-editor' => 'A Wikia fan',
```

Comment:
@mklucsarits ok, I'm going to let this one be for this ticket and not confuse things.